rotkehlchen/chain/evm/decoding/thegraph/decoder.py

```diff
@@ -58,6 +63,51 @@ def _decode_delegator_staking(self, context: DecoderContext) -> DecodingOutput:
             return self._decode_stake_locked(context)
         elif context.tx_log.topics[0] == TOPIC_STAKE_DELEGATED_WITHDRAWN:
             return self._decode_stake_withdrawn(context)
+        elif context.tx_log.topics[0] == DELEGATION_TRANSFERRED_TO_L2:
+            return self._decode_delegation_transferred_to_l2(context)
+        return DEFAULT_DECODING_OUTPUT
+
+    def _decode_delegation_transferred_to_l2(self, context: DecoderContext) -> DecodingOutput:
+        """Decode a transfer delegating GRT from ethereum to arbitrum"""
+        delegator_l2 = hex_or_bytes_to_address(context.tx_log.topics[2])
```

Comment:
Shouldn't you check if any of these addresses are tracked? Who do we care if they are tracked? Otherwise you end up creating useless events for addresses we don't care about.